.github/workflows/scorecards-analysis.yml

```
@@ -48,6 +48,6 @@ jobs:
 
       # Upload the results to GitHub's code scanning dashboard.
       - name: "Upload to code-scanning"
-        uses: github/codeql-action/upload-sarif@acb9cb18eec7e3a113ef83cff0be91e75cfd9526 # v2.1.19+patch
+        uses: github/codeql-action/upload-sarif@ 396bb3e45325a47dd9ef434068033c6d5bb0d11a # 3.27.3
```

🙅🏻 FIX AUOT-SPACE TYPO

```suggestion
        uses: github/codeql-action/upload-sarif@396bb3e45325a47dd9ef434068033c6d5bb0d11a # 3.27.3
```